tests/components/mqtt/test_device_tracker.py

```diff
@@ -403,16 +403,12 @@ async def test_setting_device_tracker_location_via_lat_lon_message(
     mqtt_mock_entry: MqttMockHAClientGenerator,
     caplog: pytest.LogCaptureFixture,
 ) -> None:
-    """Test the setting of the latitude and longitude via MQTT."""
+    """Test the setting of the latitude and longitude via MQTT without state topic."""
```

Comment:
Should we add a new test instead of modifying an existing one?

Reply:
We have another test that uses a combination of state and GPS updates and shows how to reset the override (`test_setting_device_tracker_location_via_reset_message`), the one updated one did not use a `state_topic`, so it was most ideal to update the test.